### PR TITLE
Added Auth to Recursive Builder

### PR DESCRIPTION
**Pull request by NateDiamond**
This works with the `default` instance in Auth. I think we would want to make a separate instance for non-development and get the environment variables updated in the production code. I did not push `.env.local`.

### CODE REVIEW

**Review by thaddeusdiamond — APPROVED** [review]
This should be good to go, we'll just need to update the env variables on Vercel when we push it.

**Comment by thaddeusdiamond** [comment]
@NateDiamond now that I merged in #2 try to rebase this and rebuild it.  Looks like there are a couple quick conflicts to resolve.

File: src/app/layout.js [hunk 1]
```diff
@@ -28,6 +33,10 @@ function Header() {
             <h1 className="text-2xl text-white font-custom-titles w-72 md:w-full">Recursive Ordinals Builder</h1>
           </div>
           <div className="flex-shrink-0">
+            <a>{user ? `Hi, ${user.name}! ` : ""}</a>
```
Comment (thaddeusdiamond) [inline src/app/layout.js]: Can you use the SimpleButton component below?

import { SimpleButton } from '../components/buttons.jsx';

<SimpleButton label={isLoading ? "..." : (user ? "Login" : "Logout")} onClick={() => window.open(`/api/auth/${user ? "logout" : "login"})} active={!isLoading} />

**Review by thaddeusdiamond — CHANGES_REQUESTED** [review]
This looks good I would just update to use SimpleButton instead of making your own button type/classes...

File: src/app/layout.js (hunk 1 above)
Comment (NateDiamond) [reply src/app/layout.js]: Done! Note that this requires adding an outline to the button since it's in the header.

**Comment by thaddeusdiamond** [comment]
I like it! Merging.

**Pull request merged**